### ISSUE TITLE
Scope clear not work, the value is not cleaned.

### ISSUE THREAD

**Issue by Piquinikis** [brianpkelley/md-color-picker#97]
This related with #55, clear button not working for me.

Paste here my html code:

`<div md-color-picker label="Primary" ng-model="vm.baseModel.colors.primary"></div>`

Input is cleared but the model does not change, maintains the same value.

**Comment by nikhil-uc** [comment]
+1

**Comment by madza91** [comment]
I just saw someone has been committed fix for this issue. It is in Pull requests, but I can get it work with it...

**Comment by diosney** [comment]
Yes, same issue here, I was just about to send a PR when found out that there is already one.

@brianpkelley when do you plan to update the npm version with this? This is forcing users to use their forks.
